codacy/values-production.yaml

```
@@ -363,3 +292,69 @@ minio:
   persistence:
     enabled: true
     size: 20Gi
+
+github: {}
```

This block is missing some indentation I believe:

```suggestion
  github: {}
```

---

It was properly indented, but the following ones were not. 👍